crates/pathfinder/src/rpc/v02/method/add_deploy_account_transaction.rs

```diff
@@ -0,0 +1,132 @@
+use anyhow::Context;
+use serde_with::serde_as;
+
+use crate::rpc::serde::{FeeAsHexStr, TransactionVersionAsHexStr};
+use crate::{
+    core::{
+        CallParam, ClassHash, ContractAddress, ContractAddressSalt, Fee, StarknetTransactionHash,
+        TransactionNonce, TransactionSignatureElem, TransactionVersion,
+    },
+    rpc::v02::RpcContext,
+    sequencer::ClientApi,
+};
+
+const fn transaction_version_zero() -> TransactionVersion {
+    TransactionVersion(web3::types::H256::zero())
+}
+
+#[serde_as]
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields)]
+pub struct DeployAccountTransaction {
+    // Fields from BROADCASTED_TXN_COMMON_PROPERTIES
+    #[serde_as(as = "TransactionVersionAsHexStr")]
+    #[serde(default = "transaction_version_zero")]
+    pub version: TransactionVersion,
+    #[serde_as(as = "FeeAsHexStr")]
+    pub max_fee: Fee,
+    pub signature: Vec<TransactionSignatureElem>,
+    pub nonce: TransactionNonce,
+
+    // Fields from DEPLOY_ACCOUNT_TXN_PROPERTIES
+    pub contract_address_salt: ContractAddressSalt,
+    pub constructor_calldata: Vec<CallParam>,
+    pub class_hash: ClassHash,
+}
```

Comment:
Please move this type into `rpc::v02::types::request` and rename it to `BroadcastedDeployAccountTransactions`. It needs to be added to the `BroadcastedTransaction` enum, too.

Reply:
Done